client/src/components/Loading/index.js

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import "./style.css";
+
+export default ({ className }) => {
```

Comment:
Why is the className here ? 
I think it will be standard for all pages, am I right?

Reply:
We'd only need two if we want one for the popup. It would need to be smaller I think. What do we think?

Reply:
ok, that's okay
just add a new line after importing the react